src/containers/auth/forgotPassword/index.jsx

```diff
@@ -0,0 +1,162 @@
+/**
+ * TODO: import prop-types package
+ */
+// import PropTypes from 'prop-types'
+import React, { PureComponent } from 'react';
+import Modal from 'react-modal';
+import { FaTimes } from 'react-icons/fa';
+import { IoIosCheckmarkCircle } from 'react-icons/io';
+
+import styles from './styles.module.scss';
+import { forgotPasswordRequest } from '../../../services/auth';
+import validateAuth from '../../../validation/auth';
+import Spinner from '../../../components/spinner/index';
+
+Modal.setAppElement('#root');
+
+class ForgotPassword extends PureComponent {
+  state = {
+    status: null,
+    data: null,
+    email: null,
+    message: null,
+    pending: false,
+  };
+
+  refInput = React.createRef();
+
+  send = async email => {
+    const errors = validateAuth({ email });
+    this.setState({
+      pending: true,
+    });
+    const response = !errors.email && (await forgotPasswordRequest({ email }));
+    const { data, status } = !errors.email && response;
+    let answer =
+      !errors.email &&
+      this.setState({
+        status,
+        data,
+      });
+    answer =
+      errors.email &&
+      this.setState({
+        message: errors.email,
+      });
```

Comment:
How this answer variable works? You declare it, give it value and then replace this value?

Reply:
It maid to break notifications from Eslint